src/pudl/transform/nrelatb.py

```diff
@@ -0,0 +1,300 @@
+            "fixed o&m",
+            "fuel",
+            "lcoe",
+            "variable o&m",
+            "fcr",
+            # 2023 only params
+            "additional occ",
+            "cfc",
+            "gcc",
+            "heat rate",
+            "heat rate penalty",
+            "net output penalty",
+            "occ",
+        ],
+    )
+    unstack_tables = Unstacker(
+        rate_table=unstacker_rate_table,
+        scenario_table=unstacker_scenario_table,
+        tech_detail_table=unstacker_detail_table,
+    )
+    return unstack_tables
+
+
+@asset
+def _core_nrelatb__transform_start(raw_nrelatb__data):
+    """Transform raw NREL ATB data into semi."""
+    ## EARLY CLEANING/DEDUPING
+    raw_nrelatb__data["mystery_code"] = raw_nrelatb__data.core_metric_key.str[0]
+    raw_nrelatb__data = raw_nrelatb__data.replace("*", pd.NA)
+    raw_nrelatb__data = helpers.simplify_strings(
+        raw_nrelatb__data, ["core_metric_parameter"]
+    ).convert_dtypes()
+
+    # Remove some duplicate values. they are complete dupes
+    dupe_mask = raw_nrelatb__data.duplicated(subset=IDX_ALL, keep=False)
+    # check if all of the values in the comp
+    assert all(
+        raw_nrelatb__data[dupe_mask]
+        .groupby(by=IDX_ALL, dropna=False)[["value"]]
+        .nunique()
+        .value
+        == 1
+    )
+    # we are droping dropping records that are completely dupes - not just
+    # dupes based on IDX_ALL
+    nrelatb = pd.concat(
+        [
+            raw_nrelatb__data[~dupe_mask],
+            raw_nrelatb__data[dupe_mask].drop_duplicates(keep="first"),
+        ]
+    )
+    assert not any(nrelatb.duplicated(IDX_ALL))
+
+    # ensure that we have the same set of parameters in the unstackers
+    params_found = set(nrelatb.core_metric_parameter.unique())
+    if param_differences := params_found.symmetric_difference(
+        set(unstacker_of_tables().params_all)
+    ):
+        raise AssertionError(
+            "Found different parameters in the raw NREL ``core_metric_parameter`` column "
+            f"compared to the params in the TableUnstackers: {param_differences=}"
+        )
+    return nrelatb
+
+
+def transform_unstack(
+    nrelatb: pd.DataFrame, table_unstacker: TableUnstacker
+) -> pd.DataFrame:
+    """Unstack."""
+    mask = nrelatb.core_metric_parameter.isin(table_unstacker.params)
+    assert not nrelatb[mask].duplicated(subset=table_unstacker.idx).any()
+
+    nrelatb_unstacked = (
+        nrelatb[mask]
+        .set_index(table_unstacker.idx)
+        .sort_index()[["value"]]
+        .unstack(
+            level="core_metric_parameter",
+        )
+    )
+    nrelatb_unstacked.columns = nrelatb_unstacked.columns.droplevel()
+    # this next step regarding the fcr param is only applicable to
+    # the table w/ the fcr param:
+    if "fcr" in nrelatb_unstacked:
+        nrelatb_unstacked = broadcast_fcr_param_across_tech_detail(
+            nrelatb_unstacked, table_unstacker.idx_unstacked
+        )
```

Comment:
this could be done in `_core_nrelatb__yearly_technology_detail` bc it only is done on that table but it felt more reasonable to put it here bc it is a required second-step for unstacking